easybuild/easyconfigs/g/GDAL/GDAL-2.2.3-intel-2018a-Python-2.7.14-HDF5-1.10.1.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.3'
+hdf5_ver = '1.10.1'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s' % hdf5_ver
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+#checksums = ['eb25d6ee85f4f5ac1d5581958f8c6eed9b1d50746f82866fe92e507541def35b']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('netCDF', '4.6.0', '-HDF5-%s' % hdf5_ver),
```

Comment:
@SimonPinches `netCDF-4.6.0-intel-2018a.eb` also uses `HDF5-1.10.1`, so this is a bit silly?

Reply:
Agreed - I did this quite a while ago so maybe it didn't exist then, or I was trying 1.10.2.  Can't remember now...